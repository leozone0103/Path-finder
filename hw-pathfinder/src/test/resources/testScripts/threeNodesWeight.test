# Test findPath in a three-nodes map where the shorties path contains two segments.
CreateGraph graph

AddNode graph A
AddNode graph B
AddNode graph C

AddEdge graph A B 1.000
AddEdge graph A C 3.000
AddEdge graph B C 1.000

FindPath graph A C