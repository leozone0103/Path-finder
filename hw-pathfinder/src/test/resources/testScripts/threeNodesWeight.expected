# Test findPath in a three-nodes map where the shorties path contains two segments.
created graph graph

added node A to graph
added node B to graph
added node C to graph

added edge 1.000 from A to B in graph
added edge 3.000 from A to C in graph
added edge 1.000 from B to C in graph

path from A to C:
A to B with weight 1.000
B to C with weight 1.000
total cost: 2.000